internal/pkg/skuba/addons/gangway.go

```diff
@@ -121,10 +113,9 @@ data:
     certFile: /etc/gangway/pki/tls.crt
 
     clientID: "oidc"
-    clientSecret: "{{.GangwayClientSecret}}"
     usernameClaim: "email"
     apiServerURL: "https://{{.ControlPlaneHostAndPort}}"
-    cluster_ca_path: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+    clusterCAPath: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
wow, this was what we missed before. good catch.

Reply:
The default value is `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`, so I think it's always right because we have default value in it.

But it's good to adjust the naming.
- for snake case variable: read from an environment variable
- for camel case variable: read from config